cns/restserver/util.go

```diff
@@ -350,11 +359,145 @@ func (service *HTTPRestService) removeToBeDeletedIPStateUntransacted(
 	return 0, ""
 }
 
+func (service *HTTPRestService) getNetworkContainersResponse(
+	req cns.GetNetworkContainerRequest,
+) (getNetworkContainersResponse []cns.GetNetworkContainerResponse) {
+	var (
+		networkContainerID          string
+		getNetworkContainerResponse cns.GetNetworkContainerResponse
+		exists                      bool
+		waitingForUpdate            bool
+	)
```

Comment:
Pre-declaring variables like this is generally frowned-upon in Go (even though there's a lot of old code around that does this). This causes situations where you use one of these variables, but forget to assign to it (by later deleting the assignment). The compiler would catch that immediately ordinarily, but not if you did this C-style pre-declaration. The zero value would sneak out.